substrate/frame/support/src/traits/storage.rs

```diff
@@ -59,8 +59,33 @@ pub trait StorageInstance {
 	/// Prefix of a pallet to isolate it from other pallets.
 	fn pallet_prefix() -> &'static str;
 
+	/// Return the prefix hash of pallet instance.
+	/// NOTE: This hash must be `twox_128(pallet_prefix())`
+	/// Should not impl this function by hand. Only use the default or macro generated impls.
+	fn pallet_prefix_hash() -> [u8; 16] {
```

Comment:
Could also be named `pallet_prefix_twox128` to be explicit about the hash, but it should be fine either way.

Reply:
Yeah, now better to keep the old naming consistency.